packages/core/useGamepad/index.ts

```diff
@@ -59,6 +35,69 @@ export function mapGamepadToXbox360Controller(gamepad: Ref<Gamepad | undefined>)
   })
 }
 
+/**
+ * Maps a [standard gamepad](https://w3c.github.io/gamepad/#remapping) to a PS5 DualSense Controller.
+ *
+ * You can find more about PS5 DualSense controller's original part names at
+ * https://controller.dl.playstation.net/controller/lang/en/DS_partnames.html.
+ */
+export function mapGamepadToPS5DualSenseController(gamepad: Ref<Gamepad | undefined>) {
+  return computed(() => {
+    if (gamepad.value) {
+      return {
+        ...mapGamepadToStandardController(gamepad.value),
+        buttons: {
+          '⨯': gamepad.value.buttons[0],
+          '○': gamepad.value.buttons[1],
+          '□': gamepad.value.buttons[2],
+          '△': gamepad.value.buttons[3],
+          'L1': gamepad.value.buttons[4],
+          'R1': gamepad.value.buttons[5],
+          'L2': gamepad.value.buttons[6],
+          'R2': gamepad.value.buttons[7],
+        },
+        create: gamepad.value.buttons[8],
+        options: gamepad.value.buttons[9],
```

Comment:
IMO `create`/`options` and (`back`/`start` for XBOX) should be inside `buttons` but that will be a breaking change and doing this only in `mapGamepadToPS5DualSenseController` will cause inconsistency.